webmap/src/control/ContextMenuControl.ts

```diff
@@ -69,7 +91,10 @@ export default class ContextMenuControl extends L.Control {
             switch (item) {
                 case ContextMenuItemType.copyCoords:
                     items.set('copyCoords', {
-                        label: `${this._pl3xmap.settings!.lang.contextMenu.copyCoords} ${coords}`,
+                        label: this._pl3xmap.settings!.lang.contextMenu.copyCoords
+                            .replace(/<x>/g, "" + x)
```

Comment:
Is '`"" + x` 'an attempt to convert the number to string? Or what is it for?

You can use `Object.prototype.toString()`.

Reply:
Technically the same thing :shrug: I've replaced it with `String(value?: any)`